pkg/projects/descriptions.json

```diff
@@ -20,8 +25,23 @@
         "CommonTypes": [ ]
     },
     {
+        "Name": "Microsoft.NETCore",
+        "Description": "Provides a set of packages that can be used when building portable libraries on .NETCore based platforms.",
+        "CommonTypes": [ ]
+    },
+    {
         "Name": "Microsoft.NETCore.App",
         "Description": "A set of .NET API's that are included in the default .NET Core application model.",
         "CommonTypes": [ ]
     },
+    {
+        "Name": "Microsoft.NETCore.Console",
```

Comment:
No, this was a meta-package I made like a year or more ago when @KrzysztofCwalina was first doing the console-app-model prototype.  It's sense been used in various demos / test scenarios.  Its purpose was to include every package that was supported x-plat.  I think we can phase it out once we clean up consumers: https://github.com/search?utf8=%E2%9C%93&q=%22Microsoft.NETCore.Console%22&type=Code&ref=searchresults